include/CppUTest/UtestMacros.h

```diff
@@ -142,6 +142,14 @@
     UtestShell::getCurrent()->assertLongsEqual((long)0, (long)0, NULL, file, line); \
   } }
 
+#define CHECK_RELATION(first, relop, second)\
+ { SimpleString conditionString = "evaluates to ";\
```

Comment:
The message printed with this string is a bit too scarce... Maybe "The actual comparison check evaluates to \"" + ... + "\"" would be more informative to the user. 

Anyway, using the text message to handle this additional information is not good for me, the text message should be reserved for use by the user. It would be better if you created a new subclass of TextFailure and a new method assertComparison() in UTestShell to pass the actual parameter evaluation, such that it's printed in the failure string and not in the user message.

You should then also add a xxx_TEXT macro that takes a user message.